Real_Estate/models/real_estate_property.py

```diff
@@ -0,0 +1,37 @@
+from odoo import fields, models
+
+
+class Properties(models.Model):
+    _name = "real.estate.property"
+    _description = "Real Estate Property Table to store information."
+
+    name = fields.Char('Property Name', required=True)
+    description = fields.Text('Property Description')
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date('Date Availability', copy=False,
+                                    default=lambda self: fields.Date.today() + relativedelta(months=3))
```

Comment:
```suggestion
    date_availability = fields.Date(
        'Date Availability',
        copy=False,
        default=lambda self: fields.Date.today() + relativedelta(months=3)
    )
```
Make sure to have a readable code